.github/workflows/manual-artifacts-generation.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  workflow_dispatch:
+    inputs:
+      integrationName:
+        description: 'The integration (exporter) to compile and package. Eg: "powerdns"'
+        required: true
+        default: 'powerdns'
+        type: string
+env:
+  OHAI_PFX_CERTIFICATE_BASE64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }} # base64 encoded
+  OHAI_PFX_PASSPHRASE:  ${{ secrets.OHAI_PFX_PASSPHRASE }}
+
+name: Check ${{ inputs.integrationName }} artifacts
```

Comment:
Could we remove that `${{ inputs.integrationName }}`?

We should have a proper naming for it to be able to find it faster in the workflows:
![Screenshot 2022-12-21 at 14 05 07](https://user-images.githubusercontent.com/53659978/208912275-56677d3f-587d-4df3-a816-78db4e36cd58.png)

Reply:
Updated here: [15fd6fe](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/143/commits/15fd6feb9139543f08c3a3a6893b9866f165f5cf)